tests/blackbox_test.go

```diff
@@ -43,6 +45,8 @@ var (
 	testTimeout = time.Second * 60
 	// somewhat of an abuse of contexts but go's got our hands tied
 	killContext = context.TODO()
+
+	runPattern = flag.String("blackbox.run", "", "run only blackbox tests matching regexp")
```

Comment:
This can already be done natively with the golang test runner. You can specify a particular test with something like ` -test.run  TestIgnitionBlackBox/Create_a_Directory_on_the_Root_Filesystem_2.3.0`. This commit can be dropped. 

cc @arithx since he was curious if this was possible.

Reply:
@ajeddeloh do patterns work though? Because this will allow for regexp pattern matching against subtest names as well.

Reply:
Yeah, or at least globbing does. I can throw a * at the end to get all the versions.

Reply:
Looks like this got squashed into the test commit.

Reply:
Ugh, thank you.